apps/api/src/app/user/mappers/user.mapper.ts

```diff
@@ -0,0 +1,28 @@
+import {RegisterUserDto} from "../dtos/register-user.dto";
+import {UserModel} from "../models/user.model";
+import {UserDto} from "../dtos/user.dto";
+
+
+export class UserMapper{
+
+  static mapCreateUserToModel(dto:RegisterUserDto): UserModel{
+    return new UserModel({
+      id:undefined,
+      name:dto.name,
+      email:dto.email,
+      roles:dto.roles,
+      password:dto.password,
```

Comment:
**issue:** take care that the password is not saved in plaintext. Use `bcrypt` to hash it